yelpInfo.py

```diff
@@ -21,7 +19,7 @@
 from urllib.parse import quote
 from urllib.parse import urlencode
 
-API_KEY = os.environ.get("YELP_API_KEY", None)
+API_KEY = os.environ.get("YelpAPIKey")
```

Comment:
At some point we need to standardized what we are calling these variables because I think we all have slightly different key names - not a right now issue just a comment I wanted to make while I thought of it